### PR TITLE
Add List Meetings Feature

### LINKED ISSUE #126: Add List Meetings Feature

### PR DESCRIPTION
**Pull request by apollo-tan**
Fixes #126

### CODE REVIEW

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -217,18 +210,26 @@ public void getFilteredPropertiesList_modifyList_throwsUnsupportedOperationExcep
...
 
     @Test
-    public void getIsDisplayClientsProperty_returnsBooleanPropertyType() {
+    public void getFilteredMeetingsList_modifyList_throwsUnsupportedOperationException() {
+        assertThrows(UnsupportedOperationException.class, () -> logic.getFilteredMeetingList().remove(0));
+    }
+
+    @Test
+    public void getIsDisplayClientsProperty_returnsObjectPropertyType() {
         // Call the method
```
Comment (jolwnn) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:219]: hey, just a nit, not important, but I think this comment is not necessary

File: src/test/java/seedu/address/logic/commands/ListBuyersCommandTest.java [hunk 2]
```diff
@@ -48,4 +48,34 @@ public void testExecuteBuyers() throws CommandException {
...
         assertEquals("buyers", ListBuyersCommand.KEY_WORD);
     }
+
+    @Test
+    void equals_sameCommand_returnsTrue() {
+        // Arrange
+        ListBuyersCommand listBuyersCommand1 = new ListBuyersCommand();
+        ListBuyersCommand listBuyersCommand2 = new ListBuyersCommand(); // Same command
+
+        // Act & Assert
```
Comment (jolwnn) [inline src/test/java/seedu/address/logic/commands/ListBuyersCommandTest.java:58]: Same as the previous comment, I feel that these comments don't need to be added :)

**Review by jolwnn — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 1 above)
Comment (apollo-tan) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:219]: ohh HAHAHA will look into that! Thanks for pointing out!

**Pull request merged**